test/support/generator_test_helpers.rb

```diff
@@ -17,13 +17,24 @@ def provide_existing_initializer_file
     copy_to_generator_root("config/initializers", "shopify_app.rb")
   end
 
+  def provide_existing_initializer_file_with_webhooks
+    copy_to_generator_root("config/initializers", "shopify_app_with_webhooks.rb", rename: 'shopify_app.rb')
+  end
+
   private
 
-  def copy_to_generator_root(destination, template)
+  def copy_to_generator_root(destination, template, rename: nil)
```

Comment:
this rename flag lets us have a couple of templates for a file that will be modified by the generator.